### PR TITLE
Only export include directories if installing

### PR DESCRIPTION
**Pull request by esteve**

### CODE REVIEW

File: rosidl_typesupport_opensplice_c/cmake/rosidl_typesupport_opensplice_c_generate_interfaces.cmake [hunk 1]
```diff
@@ -284,9 +284,8 @@ if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
...
     RUNTIME DESTINATION bin
   )
-ament_export_libraries(${rosidl_generate_interfaces_TARGET}${_target_suffix} ${OpenSplice_LIBRARIES})
-
-endif()
 
+  ament_export_libraries(${rosidl_generate_interfaces_TARGET}${_target_suffix} ${OpenSplice_LIBRARIES})
 
-ament_export_include_directories(include)
+  ament_export_include_directories(include)
```
Comment (dirk-thomas) [inline rosidl_typesupport_opensplice_c/cmake/rosidl_typesupport_opensplice_c_generate_interfaces.cmake]: This should only happen when the package actually installs headers.

Same below.

Comment (esteve) [inline rosidl_typesupport_opensplice_c/cmake/rosidl_typesupport_opensplice_c_generate_interfaces.cmake]: So should I remove this? Or put it elsewhere?

Comment (dirk-thomas) [inline rosidl_typesupport_opensplice_c/cmake/rosidl_typesupport_opensplice_c_generate_interfaces.cmake]: Look further up where and when headers are being installed to the include folder. The condition is different. It's probably easiest to do it there.

Comment (esteve) [inline rosidl_typesupport_opensplice_c/cmake/rosidl_typesupport_opensplice_c_generate_interfaces.cmake]: Thanks, does this look ok? https://github.com/ros2/rmw_opensplice/commit/018e3922102409dd6b51916c2ce034bc33c36c16

Comment (dirk-thomas) [inline rosidl_typesupport_opensplice_c/cmake/rosidl_typesupport_opensplice_c_generate_interfaces.cmake]: No, the logical operator needs to be `OR` not `AND`.

Comment (esteve) [inline rosidl_typesupport_opensplice_c/cmake/rosidl_typesupport_opensplice_c_generate_interfaces.cmake]: Fixed.

**Comment by wjwwood** [comment]
Can we go ahead and consider this for merge? The changes lgtm and even without the Python stuff, these changes are correct and worth merging, imo.

**Comment by dirk-thomas** [comment]
LGTM

**Pull request merged**